domain/robstoll/atrium-domain-robstoll-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/creating/charsequence/contains/creators/CharSequenceContainsAssertionsImpl.kt

```diff
@@ -33,8 +33,8 @@ class CharSequenceContainsAssertionsImpl : CharSequenceContainsAssertions {
 
     override fun <T : CharSequence> regex(
         checkerOption: CharSequenceContains.CheckerOption<T, NoOpSearchBehaviour>,
-        expected: List<String>
-    ): AssertionGroup = _containsRegex(checkerOption, expected)
+        expected: List<Regex>
+    ): AssertionGroup = _containsRegex(checkerOption, expected.map { it.toString() })
```

Comment:
`it.pattern` but you need to adopt _containsRegex in a next step anyway. Thus you can also leave it this way